cms-adapter/src/Rsbc.Dmf.CaseManagement/CaseManager.cs

```diff
@@ -2470,7 +2494,7 @@ public async Task<ResultStatusReply> UpdateCleanPassFlag(CleanPassRequest reques
                         {
                             await dynamicsContext.LoadPropertyAsync(document, nameof(document.dfp_DocumentTypeID));
                    
-                            if (document.dfp_DocumentTypeID != null && document.statecode == 0)
+                            if (document.dfp_DocumentTypeID != null && document.statecode == 0 && document.dfp_submittalstatus == 100000009)
```

Comment:
I have concerns about this extra condition.  We should review it in person and discuss if it is necessary.